app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ProfilesController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def show_me
+    @user = User.find(current_user.id)
+    render :show
```

Comment:
I'd prefer not to have two different but exactly equal pages for one content (`/profiles/me` and `/profiles/<my-user-id>`), but instead suggest to use redirects to lead the client.

Example:

```
def show_me
    redirect_to profile_path(current_user)
end
```

Reply:
implemented in https://github.com/hpi-swt2/bookkeeper-portal-red/pull/44/commits/30f3e8fb4cc7dea36d12bb49d696821e5efb476b